src/classes/PMT_PaymentWizard_CTRL.cls

```diff
@@ -562,11 +667,33 @@ public with sharing class PMT_PaymentWizard_CTRL {
      */
     public String getCurrencySymbol() {
         if (currencySymbol == null) {
-            currencySymbol = UTIL_Currency.getInstance().getCurrencySymbol(currentOpp.Id);
+            if (UserInfo.isMultiCurrencyOrganization()) {
+                currencySymbol = (String) currentOpp.get('CurrencyIsoCode') + ' ';
```

Comment:
@patrick-yan-sf  I'm not quite sure about this. If Multi-currency is enabled, it uses the currency code as the symbol (ex: USD). However if multi-currency is not enabled in the org, it uses the UTIL_Currency() to get the actual currency symbol ("$") for the default currency of the org.

Reply:
@force2b Yes that is how the currency field works.  If you don't have multi currency enabled it displays $ or the appropriate symbol. But if you have multi currency enabled then it displays USD.  Do you want to simplify it and just use the symbol.   I think you are correct in that there could be a multi-currency scenario where it is not formatted correctly.

Reply:
@patrick-yan-sf. I see. The page is behaving like other pages where it shows the currency code next to the amount so the User knows the currency. Though the code looked odd, it's definitely correct this way. For example USD and CAD both share the "$" symbol, so a User wouldn't easily know the currency by just the symbol. Thanks for clarifying. Sorry for the false alarm.